specification/app/resource-manager/Microsoft.App/preview/2024-10-02-preview/ConnectedEnvironmentsDaprComponents.json

```
@@ -330,41 +330,29 @@
       "type": "object",
       "allOf": [
         {
-          "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ProxyResource"
+          "$ref": "./CommonDefinitions.json#/definitions/DaprComponent"
         }
       ],
       "properties": {
-        "properties": {
-          "description": "Dapr component properties",
-          "type": "object",
-          "allOf": [
-            {
-              "$ref": "./CommonDefinitions.json#/definitions/DaprComponentProperties"
-            }
+        "provisioningState": {
+          "description": "Provisioning state of the Connected Environment Dapr Component.",
+          "enum": [
+            "Succeeded",
+            "Failed",
+            "Canceled",
+            "InProgress"
```

I should have left this alone. Pls try
```suggestion
        "properties": {
          "description": "Dapr component properties",
          "type": "object",
                    "properties": {
            "provisioningState": {
              "description": "Provisioning state of the Connected Environment Dapr Component.",
              "enum": [
                "Succeeded",
                "Failed",
                "Canceled",
                "InProgress"
              ],
              "type": "string",
              "readOnly": true,
              "x-ms-enum": {
                "name": "ConnectedEnvironmentDaprComponentProvisioningState",
                "modelAsString": true
              }
            },
            "deploymentErrors": {
              "description": "Any errors that occurred during deployment or deployment validation",
              "type": "string",
              "readOnly": true
```